.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
I'm confused why this uses `pip` and not `poetry` (although the presence of a `requirements.txt` file generated by poetry might explain most of it... perhaps that file should be git-ignored?)

Reply:
@THOR300 should the `requirements.txt` file really be checked in to the repo?

Reply:
After having a look I'm gonna say `no` and that this was an integrated from the following command that was probably used due to poetry install hanging.

`poetry export --without-hashes --format=requirements.txt > requirements.txt`

Reply:
> I'm confused why this uses `pip` and not `poetry` 

So I think this comes from poetry being built on pip, agree its confusing though!

Reply:
> should the requirements.txt file really be checked in

> I'm gonna say NO 

In that case I'll add a commit cleaning up the requirements file, as I think removing it before adding dependabot will be worthwhile for avoiding confusing it 😁